src/main/java/ch/njol/skript/expressions/ExprRepeat.java

```diff
@@ -0,0 +1,92 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import ch.njol.util.StringUtils;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Name("Repeat String")
+@Description("repeats a given string, a number of times")
+@Examples({
+	"\"Hello, World! \" repeated 2 times",
+	"nl and nl repeated 5 times"
+})
+@Since("INSERT VERSION")
+public class ExprRepeat extends SimpleExpression<String> {
+
+	static {
+		Skript.registerExpression(ExprRepeat.class, String.class, ExpressionType.COMBINED, "%strings% repeated %number% time[s]");
+	}
+
+	private Expression<String> strings;
+	private Expression<Number> count;
+
+	@Override
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		strings = (Expression<String>) exprs[0];
+		count = (Expression<Number>) exprs[1];
+		return true;
+	}
+
+	@Override
+	@Nullable
+	protected String[] get(Event event) {
+		List<String> stringList = new ArrayList<>();
+		int count = this.count.getOptionalSingle(event).orElse(0).intValue();
+		for (String string : this.strings.getArray(event)) {
+			if (count < 1) {
+				stringList.add(string);
+				continue;
+			}
+			stringList.add(StringUtils.multiply(string, count));
```

Comment:
I just looked at this method, and it will just return the string itself if it's 1. So that doesn't reflect the syntax as 1 means two of the strings list. So I think a simple fix would be to increment the count int when it's defined.
```java
int count = this.count.getOptionalSingle(event).orElse(0).intValue() + 1;
```
As repeated 2 times will mean three copies are returned based on the understanding of the syntax statement.

Reply:
Alright so after testing this in my build, I don't know how I feel about it adding that +1 
when I normally read `"1" repeated 5 times` I imagine I'd be sent `"11111"` as-
```.diff
0 -> 1 -> 2 -> 3 -> 4 -> 5
1 -> 2 -> 3 -> 4 -> 5 -> 6
```
-would make me feel like we're starting at `0` not `1`

Reply:
The syntax here could be compared to using `5 times` expression with the fact we send 1-5
starting at 0 and adding up to 5

at least that's the best example I could think of when trying to explain more

Reply:
ok so then the check should be <= 1 and also the syntax doesn't need time[s] just times, 1 shouldn't be allowed for confusion.

Reply:
Should I add a `Skript.warning()` when using 1 or less?

Reply:
I wouldn't, you can just default 1 and everything under it to be the exact copy of what they inserted in the strings expression, that makes the most sense.

Reply:
People would not like the warning, because if it's a loop or a mistake insert like during a loop, it shouldn't print a warning in console.

Reply:
alright should be able to add the commit shortly then
should I also change `orElse` to 1 instead of 0?